src/groups/mqb/mqbblp/mqbblp_rootqueueengine.h

```diff
@@ -458,6 +463,15 @@ class RootQueueEngine BSLS_KEYWORD_FINAL : public mqbi::QueueEngine {
     /// THREAD: This method is called from the Queue's dispatcher thread.
     virtual void
     loadInternals(mqbcmd::QueueEngine* out) const BSLS_KEYWORD_OVERRIDE;
+
+    /// Log appllication subscription info for the specified `appKey` into the
+    /// specified `stream`.
+    ///
+    /// THREAD: This method is called from the Queue's
+    /// dispatcher thread.
+    virtual bsl::ostream& logAppSubscriptionInfo(
+        bsl::ostream&           stream,
+        const mqbu::StorageKey& appKey) const BSLS_KEYWORD_OVERRIDE;
```

Comment:
There is another overload of `logAppSubscriptionInfo` at line 216. Since they look similar, I would propose to place their declarations nearby.

Reply:
Put declarations nearby